data/fields/crossing/markings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "crossing:markings",
+    "type": "combo",
+    "label": "Crossing Markings"
```

Comment:
Some strings would be helpful for this field. In particular, the `zebra` value was chosen as a concession to mappers used to the terminology in Europe (and some other countries), but unlike its official definition in some European countries, here it only refers to a marking pattern, without implying other aspects of the crossing.

Reply:
Yes, strings would be definitely useful. I haven't included them in this PR already, because I didn't feel very confident choosing labels that are . Maybe you could help me out with these? This was one of my drafts, but several entries don't quite feel right to me (e.g. surface, zebra*, ladder:paired, …).

```
            "yes": "Unspecified Markings",
            "surface": "Distinct Surface",
            "zebra": "Zebra Stripes",
            "lines": "Solid Lines on Either Side",
            "ladder": "Ladder",
            "lines:paired": "Pairs of Lines on Either Side",
            "dashes": "Dashed Lines on Either Side",
            "dots": "Dotted Lines on Either Side",
            "zebra:double": "Zebra Stripes, Split in the Middle",
            "zebra:paired": "Zebra Stripes With Paired Bars",
            "zebra:bicolour": "Zebra Stripes With Alternating Colors",
            "ladder:skewed": "Skewed Ladder",
            "ladder:paired": "Ladder With Paired Bars",
            "no": "No Markings"
```

I think that eventually, it would probably be a better user interface if these options were presented using a pictorial form, like in this graphic on wikipedia: 
[![](https://upload.wikimedia.org/wikipedia/commons/2/2d/Crosswalk_styles_%28en%29.svg)](https://commons.wikimedia.org/wiki/File:Crosswalk_styles_(en).svg)

Reply:
Similar to `tracktype`, we can probably get away with longer, more descriptive strings to avoid confusion. I don’t think even the traffic engineering profession has a set term for every one of these values, let alone any dialect of colloquial English. In case it helps, here are the technical terms for these patterns in the U.S., paired with my best guess at what non-U.S. markings would be called, but I think using them without a more descriptive tooltip would risk mistranslation and misuse by lay mappers:

Value | Description
----|----
`yes` | Marked Somehow
`surface` | Surface Treatment Only
`zebra` | Longitudinal Bars
`lines` | Transverse Lines
`ladder` | Ladder With Longitudinal Bars
`lines:paired` | Double Transverse Lines
`dashes` | Dashed Transverse Lines
`dots` | Dotted Transverse Lines
`zebra:double` | Triple-Four (!?!?)
`zebra:paired` | Paired Longitudinal Bars
`zebra:bicolour` | Longitudinal Bars With Alternating Colors
`ladder:skewed` | Ladder With Diagonal Bars
`ladder:paired` | Ladder With Paired Longitudinal Bars
`no` | Unmarked